### PR TITLE
Add a basic Dockerfile

### PR DESCRIPTION
**Pull request by Xarthisius**
Needed for deployment.

### CODE REVIEW

**Review by matthewturk — APPROVED** [review]
I *think* this is right.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM cibuilds/hugo:0.80 AS builder
+
+COPY content content
+COPY static static
+COPY themes themes
+COPY resources resources
+COPY config.toml .
```
Comment (munkm) [inline Dockerfile]: This looks good to me. As long as we don't expect any new directories showing up? 

could we also do something like:
```
COPY ./ /yt-blog
WORKDIR /yt-blog
```

Or would that  affect the copy later?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+RUN hugo --minify
+
+FROM nginx:stable-alpine
+COPY --from=builder /home/circleci/project/public /usr/share/nginx/html
```
Comment (munkm) [inline Dockerfile:8]: just a noob question since I haven't done this before: are we supposed to set up a circleCI job on here then too?

**Review by munkm — APPROVED** [review]
Two questions since I'm learning!

File: Dockerfile (hunk 2 above)
Comment (Xarthisius) [reply Dockerfile:8]: nope, I'm using image that CircleCI created. That's why the user is called like that.

Comment (munkm) [reply Dockerfile:8]: Ahhh ok!

**Pull request merged**